server/src/views/frame.js

```diff
@@ -222,6 +222,7 @@ class Frame extends React.Component {
 
   clickFullPageButton(e) {
     e.preventDefault();
+    history.pushState({state: "frame"}, "", "#frame");
```

Comment:
Oh, maybe because it's not a place...

BUT, I believe we have an hashchange handler in clientglue?  And we could move all this code over there.